### DISCUSSION TITLE
[Feature] "View surrounding assets"

### DISCUSSION THREAD

**Discussion by titro** [immich-app/immich#4694] [Feature Request]
### The feature

A functionality like in Kibana "View surrounding assets" would be fine.
Perhaps you are looking for photos which belong to that scenario but the person is not on that specific photo but you know that there are photos which a person.
If you are going for example through the peoples page and open an image of a person you could directly jump into the scenario which +/- 10 photos before/after that. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by cr0n76** [comment]
I wanted to propose such a feature as well. But since it is hard to find what is the correct "surrounding" (which can vary heavily with the search / intentions of the user), I propose to solve it via hierachical tags and add automatically the folder structure and/or the date of recording to the hierachical tags. See my proposal at https://github.com/immich-app/immich/discussions/2168

**Reply by titro** [reply]
My initial thought was having this functionality which jumps to the "all assets" timeline without any filter but yeah with #2168 that would be cool :-)

**Comment by bo0tzz** [comment]
#4115
